apps/bugs_bunny/lib/bugs_bunny/clients/rabbitmq.ex

```diff
@@ -48,7 +58,11 @@ defmodule BugsBunny.RabbitMQ do
   def declare_exchange(_channel, "", _type, _options), do: :ok
 
   def declare_exchange(channel, exchange, type, options) do
-    Exchange.declare(channel, exchange, type, options)
+    case Exchange.declare(channel, exchange, type, options) do
+      :ok ->
+        Logger.info("exchange #{exchange} successfully declared")
+        :ok
```

Comment:
we need an `error` clause here, so, if we get an error we don't get a pattern match error instead
```ex
{:error, reason} ->
  {:error, reason}
```